config/locales/en.yml

```
@@ -626,8 +626,18 @@ en:
         company_name:
           label: "Company name"
           custom_error: "Enter the name of your company"
-        company_number:
-          label: "Company number (optional)"
+        company_is_uk_registered:
+          label: "Is your company registered in the UK?"
+          custom_error: "Select if your company is registered in the UK"
+          options:
+            united_kingdom:
+              label: "Yes"
+              input:
```

```suggestion
              company_number:
```

To make it easier to find and update the content for company number.

---

Fixed.  Thanks @leenagupte 👍